x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -437,6 +452,16 @@ export const update = async (
       user,
     });
 
+    const casesAndAssigneesToNotifyForAssignment = getCasesAndAssigneesToNotifyForAssignment(
+      updatedCases,
+      casesMap,
+      user
+    );
+
+    if (casesAndAssigneesToNotifyForAssignment.length > 0) {
```

Comment:
How about we move the if check inside of the `bulkNotifyAssignees` and just have it return if the passed in value is an empty array?